test/Microsoft.AspNetCore.Mvc.Razor.Host.Test/TestFiles/Output/DesignTime/_ViewImports.cs

```diff
@@ -77,6 +77,14 @@ public class TestFiles_Input__ViewImports_cshtml : global::Microsoft.AspNetCore.
         }
         ))();
         ((System.Action)(() => {
+System.Object __typeHelper = "Microsoft.AspNetCore.Mvc.Razor.TagHelpers.HeadTagHelper, Microsoft.AspNetCore.Mvc.Razor";
```

Comment:
@NTaylorMullen @rynowak should the generated code for `System.*` here all have `global::`?